### PR TITLE
Truncate SkyCoord exception if too long

### PR DESCRIPTION
**Pull request by Carifio24**
This PR looks to solve a problem where the exception message can be very long when one of the latitude angles is not with [-90°, 90°]. This error message comes from astropy (see https://github.com/astropy/astropy/issues/13994), so this PR just truncates the message if it's longer than 300 characters (adding a ... to the end so the user knows that this isn't the full exception).

### CODE REVIEW

File: glue_wwt/viewer/table_layer.py [hunk 1]
```diff
@@ -268,7 +268,12 @@ def _update_presentation(self, force=False, **kwargs):
                     coord = SkyCoord(lon, lat, unit=u.deg,
                                      frame=self._viewer_state.frame.lower()).icrs
                 except Exception as exc:
-                    self.disable(str(exc))
+                    # Truncate the exception string if it's too long
+                    disable_msg = str(exc)
+                    max_length = 300
+                    if len(disable_msg) > max_length:
+                        disable_msg = disable_msg[:max_length] + "..."
```
Comment (dhomeier) [inline glue_wwt/viewer/table_layer.py:275]:
```suggestion
                    # Truncate the Latitude exception string for long input arrays
                    disable_msg = str(exc)
                    if len(disable_msg) > 300:
                        lats = np.sort(lat)
                        disable_msg = f"{disable_msg[:66]}{lats[:2]} ... {lats[-2:]}]"
```
Might make the exception a bit more useful (especially if the values within the first 300 chars happen to be all within range). Alternatively returning the position of the out-of-range inputs as suggested for the astropy issue could be an option, but that might be left to upstream.
The check _could_ also be made more specific, as the exception in question would be a `ValueError` on setting `Latitude`.

Comment (dhomeier) [reply glue_wwt/viewer/table_layer.py:275]: There is some discussion in the proposed upstream fix astropy/astropy#13997 on the most helpful output for the message; once that is finalised we might also try to adopt the format here (obviously will be some time before can rely on the fixed Astropy release).

Comment (Carifio24) [reply glue_wwt/viewer/table_layer.py:275]: I agree, matching the upstream format seems like a good way to go. It looks to me like the format has been mostly settled there. Or is it still awaiting more discussion?

**Comment by Carifio24** [comment]
@dhomeier I've updated the message format here to give essentially the same method as your proposed upstream fix.

**Comment by Carifio24** [comment]
Was just looking through some of my old open PRs. @astrofrog @dhomeier do we think the format of this message is good now? It matches the proposed upstream change quite closely.

**Review by astrofrog — APPROVED** [review]
Looks good to me!

**Pull request merged**